.gitignore

```diff
@@ -0,0 +1,28 @@
+### Node ###
+lib-cov
+*.seed
+*.log
+*.csv
+*.dat
+*.out
+*.pid
+*.gz
+
+pids
+logs
+results
+
+npm-debug.log*
+node_modules
+
+### Linux ###
+.*
+!.gitignore
+!.git*
+!.eslint*
+!.npmignore
+!.travis.yml
```

Comment:
We would want .gitignore, .eslint, and .travis.yml updates to be noticed, wouldn't we?